platform/docs/docs/platform/services/customization-service/sampleCustomizations.tsx

```diff
@@ -1476,6 +1476,94 @@ window.config = {
       },
     },
   ],
+};
+  `,
+  },
+  {
+    id: 'studyBrowser.thumbnailDoubleClickCallback',
```

Comment:
This should go to studyBrowser section https://docs.ohif.org/platform/services/customization-service/StudyBrowser

Reply:
It will appear there. You can see that other keys such as `studyBrowser.studyMenuItems` are defined there in the same file and object.